packages/pluggableWidgets/carousel-web/src/components/__tests__/__snapshots__/Carousel.spec.tsx.snap

```diff
@@ -18,7 +18,26 @@ exports[`Carousel renders correctly 1`] = `
     />
     <div
       className="swiper-wrapper"
-    />
+    >
+      <div
+        className="swiper-slide swiper-slide-duplicate"
+        data-swiper-slide-index={0}
+      >
+        <div />
+      </div>
+      <div
+        className="swiper-slide"
+        data-swiper-slide-index={0}
+      >
+        <div />
+      </div>
+      <div
+        className="swiper-slide swiper-slide-duplicate"
+        data-swiper-slide-index={0}
```

Comment:
Should all of these have index 0?

Reply:
I believe so, this is a part of Swiper module and it should work fine.